test/src/System/Time.cpp

```diff
@@ -0,0 +1,172 @@
+#include <SFML/System/Time.hpp>
+
+#include <catch.hpp>
+
+TEST_CASE("sf::Time class", "[system]")
+{
+    SECTION("Construction")
+    {
+        SECTION("Default constructor")
+        {
+            const sf::Time time;
+            CHECK(0.0f == time.asSeconds());
+            CHECK(0 == time.asMilliseconds());
+            CHECK(0 == time.asMicroseconds());
```

Comment:
+1. All compiler will warn about using `=` by accident anyway.

Reply:
> personally I find it really unintuitive
 
 Agreed. I tend to use this style as a holdover from when I used GTest whose assertion macros look like this:
 ```cpp
 EXPECT_EQ(42, sf::someFunction());
 ```
since their two-argument form expects the first value to be the expected value and the second the actual value. You're right that I switched to the other style towards the end of the file haha. I'll switch entirely to the style you (and I) prefer so everything is consistent.

Reply:
Fixed. Thanks for pointing this out. I've been on the fence about this style for while after switching to Catch2 but I think this has made my mind that `CHECK(functionUnderTest() == expected_value)` is the better style when using single-parameter test macros.